recipes/git-bash/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "git-bash" %}
+{% set version = "2.20.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/git-for-windows/git/releases/download/v{{ version }}.windows.1/PortableGit-{{ version }}-64-bit.7z.exe   # [win64]
+  sha256: 4c0fc0c77d577f8dbb3d7711283458ba1d0ce1781cfe0d49c940abbcdf18faa0   # [win64]
```

Comment:
It doesn't seem like you are actually applying the patches.

Reply:
@scopatz Patches are applied in the bld.bat